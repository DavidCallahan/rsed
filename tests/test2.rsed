#!./rsed
print "Test 1"
skip to "more"
copy all
