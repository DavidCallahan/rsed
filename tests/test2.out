Test 1
3 more
4
5 less
6
